File: ServiceMap/src/ServiceMap/wwwroot/app/servicesTnt/serviceTnt.ts
import { IPage } from '../pagination/page';

export interface IServiceTnt {
    depotCode: string;
    town: string;
    fromPostcode: string;
    toPostcode: string;
    sobota: boolean;
    ex9: boolean;
    ex10: boolean;
    ex12: boolean;
    priority: string;
    wieczorneDostarczenie?: boolean;
    standardDeliveryOd: string;
    standardDeliveryDo: string;
    pickUpDomesticZgl: string;
    dateTimePickUpEksportSmZgl: string
    samochodZwindaDostepnyWstandardzie?: boolean;
    diplomatNextDay: string;
    serwisMiejski?: boolean;
    serwisPodmiejski?: boolean;
    pickUpDomesticCzas: string;
    pickUpEksportSmCzas: string;
}
export interface IServiceFilter {
    postCode: string;
    cityName: string;
    currentPage?: number;
}

export interface IServiceTntResult {
    serviceTnt: IServiceTnt[];
    paging: IPage[];
}

File: ServiceMap/src/ServiceMap/Models/ServiceTnt/ServiceTnt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServiceMap.Models.apiModels
{
    public class ServiceTnt
    {
        public string DepotCode1a { get; set; }
        public string Town { get; set; }
        public string FromPostcode { get; set; }
        public string ToPostcode { get; set; }
        public bool Sobota { get; set; }
        public bool EX9 { get; set; }
        public bool EX10 { get; set; }
        public bool EX12 { get; set; }
        public TimeSpan Priority { get; set; }
        public bool WieczorneDostarczenie { get; set; }
        public TimeSpan StandardDeliveryOd { get; set; }
        public TimeSpan StandardDeliveryDo { get; set; }
        public TimeSpan PickUpDomesticZgl { get; set; }
        public TimeSpan? DateTimePickUpEksportSmZgl { get; set; }
        public bool? SamochodZwindaDostepnyWstandardzie { get; set; }
        public TimeSpan? DiplomatNextDay { get; set; }
        public bool? SerwisPodmiejski { get; set; }
        public string PickUpDomesticCzas { get; set; }
        public string PickUpEksportSmCzas { get; set; }
        public bool? SerwisMiejski { get; set; }
    }   
}

